src/test/java/org/jenkinsci/plugins/workflow/cps/actions/ArgumentsActionImplTest.java

```diff
@@ -394,7 +395,7 @@ public void testUnusualStepInstantiations() throws Exception {
                 " node('master') { \n" +
                 "   writeFile text: 'hello world', file: 'msg.out'\n" +
                 "   step([$class: 'ArtifactArchiver', artifacts: 'msg.out', fingerprint: false])\n "+
-                "   withEnv(['CUSTOM=val']) {\n"+  //Symbol-based, because withEnv is a metastep
+                "   withEnv(['CUSTOM=val']) {\n"+  //Symbol-based, because withEnv is a metastep; TODO huh? no it is not
```

Comment:
Not sure what @svanoort thought he was testing in this case. If the idea was to test a symbol-based block-scoped metastep (like `configFileProvider`), it is not doing that.